iroha/src/sumeragi.rs

```diff
@@ -266,19 +289,29 @@ impl Sumeragi {
                     if leader != peer_id {
                         send_futures.push(message.clone().send_to(&leader))
                     }
-                    futures::future::join_all(send_futures).await;
+                    let results = futures::future::join_all(send_futures).await;
+                    results
+                        .iter()
+                        .filter(|result| result.is_err())
+                        .for_each(|error_result| {
+                            eprintln!("Failed to send messages: {:?}", error_result)
+                        });
                 }
             }
         });
     }
 
+    #[log]
```

Comment:
Maybe remove message `#[log]` from `handle_message` if you add it to its handler methods?

Reply:
Let it stay here till the refactoring